scope.py

```diff
@@ -278,6 +324,49 @@ def doc(self):
             print("Kowalski connection not established, cannot generate docs.")
             return
 
+        period_limits = {'cepheid': [1.0, 100.0],
+                         'delta_scuti': [0.03, 0.3],
+                         'beta_lyr': [0.3, 25],
+                         'rr_lyr': [0.2, 1.0],
+                         'w_uma': [0.2, 0.8]}
+        period_loglimits = {'cepheid': True,
+                            'delta_scuti': False,
+                            'beta_lyr': True,
+                            'rr_lyr': False,
+                            'w_uma': False}
```

Comment:
Lint, please

Reply:
What exactly is failing? I ran the pre-commit install and so on.

Reply:
I accidentally introduced a bug in the pre-commit hook config in a previous pr. Just fixed, please merge in main, linter will work then.